connect/file/src/test/java/org/apache/kafka/connect/file/FileStreamSinkTaskTest.java

```diff
@@ -19,6 +19,8 @@
 
 import org.apache.kafka.clients.consumer.OffsetAndMetadata;
 import org.apache.kafka.common.TopicPartition;
+import org.apache.kafka.common.record.Record;
```

Comment:
We'll probably need more in the way of tests for a final version -- validation that the timestamp actually gets passed along for SourceRecords, properly passed into SinkRecords, etc.